### PR TITLE
fix(ASSETS-6927):Upon invoking the “Add” button present in the page, focus is not shifted to the newly identified field.

### PR DESCRIPTION
**Pull request by ionelc-ensemble**
After adding a new input, switch focus to it immediately.

## Motivation and Context

https://jira.corp.adobe.com/browse/ASSETS-6927

## How Has This Been Tested?

Tested manually on local AEM instance in Chrome 106 on macOS 12.6

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by ionelc-ensemble** [comment]
@Pareesh @RitwikSrivastava this is ready for reviewing (I cannot add reviewers)

File: coral-component-multifield/src/scripts/Multifield.js [hunk 1]
```diff
@@ -246,6 +246,13 @@ const Multifield = Decorator(class extends BaseComponent(HTMLElement) {
         this.trigger('change');
 
         this._trackEvent('click', 'add item button', event);
+
+        // Focus the newly created input
+        const items = this.items.getAll();
+        const setsize = items.length;
+        const itemToFocus = items[setsize - 1];
+        const inputItem = itemToFocus.querySelector("coral-multifield-item-content > input");
```
Comment (Pareesh) [inline coral-component-multifield/src/scripts/Multifield.js:254]: The multifield component support any template, it's not necessary that its will always contain an input field.

Comment (majornista) [reply coral-component-multifield/src/scripts/Multifield.js:254]: You can use `commons.TABBABLE_ELEMENT_SELECTOR` to cover different types of elements.

**Comment by RitwikSrivastava** [comment]
@majornista  Can you please review this ?

File: coral-component-multifield/src/scripts/Multifield.js [hunk 2]
```diff
@@ -302,6 +302,18 @@ const Multifield = Decorator(class extends BaseComponent(HTMLElement) {
         this.trigger('change');
 
         this._trackEvent('click', 'add item button', event);
+
+        // Focus the newly created input if it can receive focus
+        const items = this.items.getAll();
+        const setsize = items.length;
+        const itemToFocus = items[setsize - 1];
+        const inputItem = itemToFocus.querySelector(commons.TABBABLE_ELEMENT_SELECTOR);
```
Comment (Pareesh) [inline coral-component-multifield/src/scripts/Multifield.js:310]: please change this variable name accordingly

**Review by majornista — APPROVED** [review]

**Comment by ionelc-ensemble** [comment]
@majornista thanks for review & approval. If possible, please merge or merge & release

**Pull request merged**